torso-bundle.js

```diff
@@ -305,6 +305,51 @@
           },
 
           /**
+           * Will force the cache to fetch a subset of this collection's tracked ids
+           * @method requesterMixin.fetchSubsetOfTrackedIds
+           * @return {Promise} promise that will resolve when the fetch is complete
+           */
+          fetchSubsetOfTrackedIds: function(ids) {
+            var subsetOfTrackedIds = _.intersection(ids, this.getTrackedIds());
```

Comment:
Is there a use case where you wouldn't want to track the missing ids? Even if so, is it common enough that we wouldn't want to default to tracking anyhow?